docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -213,22 +218,22 @@ defs = Definitions(
 )
 ```
 
-Using source assets has a few advantages over having the code inside of an asset's op load the data:
+Using source assets has a few advantages over having the code inside of an asset's function load the data without specifying any dependencies:
 
 - **The UI can show asset lineage that includes the source assets**. If different asset definitions in different code locations have the same asset key as a <PyObject object="SourceAsset" /> and both code locations are loaded into the underlying webserver, the UI can represent the asset lineage across the code locations. This can be accomplished using [workspace files](/concepts/code-locations/workspace-files).
 - **Dagster can use data-loading code factored into an <PyObject object="IOManager" /> to load the contents of the source asset**.
-- **Asset dependencies can be written in a consistent way,** independent of whether they're downstream from a source asset or a derived asset. This makes it easy to swap out a source asset for a derived asset and vice versa.
+- **Asset dependencies can be written in a consistent way,** independent of whether they're downstream from a source asset or a software-defined asset. This makes it easy to swap out a source asset for a software-defined asset and vice versa.
 
-### Graph-backed assets and multi-assets
+### Writing more computationally complex assets
```

Comment:
did a rework of this section to avoid `ops` and `graphs` so that, ideally, a new user would be able to understand what `multi_asset` and `graph backed asset` do without needing to first understand other concepts

Reply:
that's a good callout. i will think on what this could be re-titled